client/src/Controller/Admin/IndexController.php

```diff
@@ -518,4 +525,18 @@ private function dispatchCSVUploadEvent()
 
         $this->eventDispatcher->dispatch($csvUploadedEvent, CSVUploadedEvent::NAME);
     }
+
+    private function dispatchAdminManagerDeletedEvent(User $userToDelete)
+    {
+        $trigger = AuditEvents::TRIGGER_ADMIN_MANAGER_MANUALLY_DELETED;
+        $currentUser = $this->tokenStorage->getToken()->getUser();
+
+        $adminManagerDeletedEvent = new adminManagerDeletedEvent(
```

Comment:
Technically I think PHP will let this work but the class name should be capitalised:

`AdminManagerDeletedEvent`